test/e2e/far_e2e_test.go

```diff
@@ -78,71 +80,95 @@ var _ = Describe("FAR E2e", func() {
 		testNodeParam, err = buildNodeParameters(clusterPlatform.Status.PlatformStatus.Type)
 		Expect(err).ToNot(HaveOccurred(), "can't get node information")
 	})
+	Context("stress cluster with ResourceDeletion remediation strategy", func() {
+		var (
+			filteredNodes                 *corev1.NodeList
+			selectedNode                  *corev1.Node
+			nodeName                      string
+			pod                           *corev1.Pod
+			startTime, nodeBootTimeBefore time.Time
+			err                           error
+			remediationStrategy           v1alpha1.RemediationStrategyType
```

Comment:
I believe the following vars can be globally shared between the ResourceDeletion Context and OutOfServiceTaint Context.
Can you move them from here to https://github.com/medik8s/fence-agents-remediation/pull/92/files#diff-b9729ac4458c7637438e3bf558999b35adedcaf4d13af1e3eb752db501fd4997R54-R58 and remove them from OutOfServiceTaint Context.
Only `filteredNodes` is important to be set as nil per context.

Reply:
Fixed it as you suggested. Now setting filteredNodes as nil per context.

Reply:
I think you got me wrong here, as I wanted to move all the above 7 variables from ResourceDeletion Context and OutOfServiceTaint Context to their "parent Context", _Describe_ `FAR E2e`. These variables are first set (e.g., `pod = createTestedPod(nodeName, testContainerName)`) at the BeforeEach per strategy Context, and only later their values are used (e.g., `checkRemediation(nodeName, nodeBootTimeBefore, pod, remediationStrategy)`). Therefore they can be shared between the strategy Contexts, under Describe `FAR E2e`.

> Now setting filteredNodes as nil per context.

I didn't suggest moving `filteredNodes` variable to be var of Describe, but now the fourth e2e test is failing (there is no fourth worker). I would just revert and keep this variable under each strategy Context.

Reply:
Thank you for correcting me. I just kept the filteredNodes variable under each strategy Context.
The changes from the last commit is https://github.com/k-keiichi-rh/fence-agents-remediation/commit/b0c26dcc28ca18a57aa725ba052468ab91069f48.